### PR TITLE
Prepare for lnd v0.13.0 and upcoming LiT version

### PR DESCRIPTION
**Pull request by guggero**
We want to publish a version of LiT that contains `lnd v0.13.0-beta` soon. To make sure `poold` is _compile time_ ready for that version of `lnd` we need to bump the dependency version.

We also bump the daemon version to `v0.5.0-alpha` to prepare for the release of sidecar channels and self channel balance.

### CODE REVIEW

File: go.mod
Comment (Roasbeef) [inline go.mod:20]: We should be cutting this next release before 0.13 is finalzied, but I think that's ok, we'll just need to circle back and update things if any critical API level stuff has changed.

**Review by Roasbeef — APPROVED** [review]
LGTM 🦖

**Comment by guggero** [comment]
Please hold off on merging this. I think we need some fix for the "wait for lnd to be unlocked" forward compatibility for lnd `v0.13.0-beta`.

**Comment by guggero** [comment]
Updated `lndclient` to `v0.12.0-8` which fixes the "wait for lnd to be unlocked" problem.
This is ready for review and testing now.

**Review by wpaulino — APPROVED** [review]

**Comment by wpaulino** [comment]
Should hold off on merging until #259 is merged and anything else that may come up.

**Pull request merged**
